recording/lib/reporter.ex

```diff
@@ -6,6 +6,9 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Reporter do
   use GenServer
 
   alias Membrane.RTC.Engine.Track
+  alias Membrane.RTCP.SenderReportPacket
```

Comment:
Would be nice to describe what is `timestamp` in `add_track`, `start_timestamp` and `end_timestamp` and what is the general behaviour of calculating track offests. As far as I understand we only use rtcp for syncing a track when we received RTCP report for this track and we received RTCP report for the first track in the conversation